### PR TITLE
Update contributing tab navigation

### PR DESCRIPTION
**Pull request by floriographygoth**
Update contributing tab left navigation
Update contributing tab header navigation to match docs.bazel.build

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -32,12 +32,18 @@
...
+            </li>
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
+              <a href="{{ site.docs_site_url }}">Using Bazel</a>
+            </li>
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
+              <a href="{{ site.extending_category_url }}">Extending Bazel</a>
+            </li>
+            <li{% if page.nav == "contribute" %} class="active"{% endif %}>
+              <a href="{{ site.main_site_url }}/contributing.html">Contributing</a>
+            </li>
```
Comment (jin) [inline _includes/header.html:46]: These look off on the desktop:

Screen Shot 2021-03-05 at 12 36 59 PM

Comment (jin) [reply _includes/header.html:46]: Looks like some significant whitespace characters made it in. I cleaned it up, try applying the code below:
```suggestion
            <li{% if page.nav == "docs" %} class="active" {% endif %}>
                <a href="{{ site.getting_started_category_url }}">Getting Started</a>
            </li>
            <li{% if page.nav == "docs" %} class="active" {% endif %}>
                <a href="{{ site.docs_site_url }}">Using Bazel</a>
            </li>
            <li{% if page.nav == "docs" %} class="active" {% endif %}>
                <a href="{{ site.extending_category_url }}">Extending Bazel</a>
            </li>
            <li{% if page.nav == "contribute" %} class="active" {% endif %}>
                <a href="{{ site.main_site_url }}/contributing.html">Contributing</a>
            </li>
```

Comment (jin) [reply _includes/header.html:46]: Screen Shot 2021-03-05 at 12 48 06 PM

Looks fine now.

**Comment by floriographygoth** [comment]
Thank you for the review. I've accepted your updates!

**Review by jin — APPROVED** [review]

**Pull request merged**